underfs/s3/src/main/java/alluxio/underfs/s3/S3UnderFileSystem.java

```diff
@@ -414,19 +414,19 @@ public void setOwner(String path, String user, String group) {}
   @Override
   public void setMode(String path, short mode) throws IOException {}
 
-  // Returns the bucket owner.
+  // Returns the account owner.
   @Override
   public String getOwner(String path) throws IOException {
     return mAccountOwner;
   }
 
-  // No group in S3 ACL, returns the bucket owner.
+  // No group in S3 ACL, returns the account owner.
```

Comment:
Account owner and bucket owner are different. 
- Account owner is inferred from the S3 credentials that the user provides.
- Bucket owner is the AWS account who creates and owns the bucket.

We map the account owner to the owner of mounted Alluxio directory, not the bucket owner. Because Alice can access Bob's bucket and mount it to Alluxio, in this case, Alice should be the Alluxio dir/file owner.